tools/internal_ci/helper_scripts/delete_nonartifacts.sh

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env bash
+# Copyright 2017 gRPC authors.
```

Comment:
Please fix Copyright date for new file. Also we are creating new files with "The gRPC Authors" without the period at the end these days after @carl-mastrangelo fixed the copyright check script.

```
# Copyright 2018 The gRPC Authors
```